.circleci/deployment/commands.yml

```diff
@@ -37,16 +37,43 @@
           backend-appname: <<parameters.backend-appname>>
           frontend-appname: <<parameters.frontend-appname>>
           cf-space: <<parameters.cf-space>>
-      - deploy-clamav:
-          backend-appname: <<parameters.backend-appname>>
-          cf-org: <<parameters.cf-org>>
-          cf-space: <<parameters.cf-space>>
       - deploy-frontend:
           environment: <<parameters.environment>>
           backend-appname: <<parameters.backend-appname>>
           frontend-appname: <<parameters.frontend-appname>>
           cf-space: <<parameters.cf-space>>
 
+  clamav-cloud-dot-gov:
```

Comment:
Does this command only ever run for prod/master? Right, we don't want dev/staging apps to be pushing the `manifest.clamav.yml`?

Reply:
pulled all the variables that we didn;t need anymore out.